modal-style-mobile.css

```diff
@@ -0,0 +1,128 @@
+#modal {
+  display: none;
+}
+
+.activeblur {
+  filter: blur(4px);
+}
+
+.modal-section {
+  position: fixed;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 2fr 1.5fr 3.5fr 5fr 1.5fr;
+  flex-direction: column;
+  border: 1px solid black;
+  background-color: white;
+  z-index: 99;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  height: 90%;
+  width: 89%;
+  padding: 0 0.5vh 0 2.5vh;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1vh 1vh 4vh 1vh;
+  grid-area: 1 / 1 / 2 / 2;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 1.3rem;
+  line-height: 38px;
+  overflow: hidden;
+  color: #172b4d;
+}
+
+.modal-header > h2 {
+  overflow: hidden;
+}
+
+.modal-list {
+  grid-area: 2 / 1 / 3 / 2;
+  list-style-type: none;
+  display: flex;
+  flex-flow: row nowrap;
+  margin-bottom: 2.5vh;
+  color: #3c3a39;
+}
+
+.modal-list > li {
+  border: 1px solid #3c3a39;
+  margin-left: 1vh;
+  padding: 1vh;
+  overflow: hidden;
+}
+
+.modal-img {
+  display: flex;
+  grid-area: 3 / 1 / 4 / 2;
+}
+
+.modal-description {
+  grid-area: 4 / 1 / 5 / 2;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.modal-btn {
+  grid-area: 5 / 1 / 6 / 2;
+}
+
+.modal-link {
+  width: 16vh;
+  height: 46px;
+  overflow: hidden;
+}
+
+.modal-img,
+.modal-description,
+.modal-btn {
+  margin-right: 5vh;
+  margin-bottom: 1vh;
+  margin-left: 1vh;
+}
```

Comment:
- [ ] Kindly give the image in this `modal-img` div a width of 100% so that it takes up the entire width of its container and looks more like the Figma design
## Design Guideline
![image](https://user-images.githubusercontent.com/46839548/190654816-e96b4a81-4c6d-4603-9372-17ca507a7c3c.png)

## Your Implementation
![image](https://user-images.githubusercontent.com/46839548/190654655-3b83d640-871c-403b-abd4-dff39eeb3507.png)